R/load.R

```diff
@@ -842,18 +842,19 @@ renv_load_report_synchronized <- function(project = NULL, lockfile = NULL) {
       "- One or more packages recorded in the lockfile are not installed.",
       "- Use `renv::status()` for more details."
     )
-    writef(msg)
+    caution(msg)
     return(FALSE)
   }
 
   # otherwise, use status to detect if we're synchronized
   info <- local({
     renv_scope_options(renv.verbose = FALSE)
+    renv_scope_caution(FALSE)
```

Comment:
Are you sure this should be set fo `FALSE` here?

Reply:
Yes, because we want to avoid the regular "noisy" output from status in this scope -- instead, we want to ask users to run `renv::status()` themselves to see the full log.